packages/client/src/object/convertWireToOsdkObjects.test.ts

```diff
@@ -85,7 +86,7 @@ describe("convertWireToOsdkObjects", () => {
 
     // Should not have $title
     expect(JSON.stringify(employee)).toMatchInlineSnapshot(
-      `"{"employeeId":50030,"fullName":"John Doe","office":"NYC","class":"Red","startDate":"2019-01-01","employeeStatus":{},"$apiName":"Employee","$objectType":"Employee","$primaryKey":50030}"`,
+      `"{"employeeId":50030,"fullName":"John Doe","office":"NYC","class":"Red","startDate":"2019-01-01","employeeStatus":{},"employeeLocation":{},"$apiName":"Employee","$objectType":"Employee","$primaryKey":50030}"`,
```

Comment:
Hmm. Is this how we want this value to serialize?

Reply:
Mmm, I just did parity with how normal time series does it. We could serialize it as the trackId and integration RID perhaps? Like that's what we get over the wire, though then I feel like timeseries should also be updated to have the timeseries ID

Reply:
If we feel strongly about this, can do this as a FLUP

Reply:
The case I want to think about is nextjs, where you load the thing on the backend and then send it to the frontend by serializing it. I think this is okay especially if we add last fetched. Honestly, I don't think nextjs is going to be a good platform for something that wants live updates.